website/docs/d/datasource_google_service_account_id_token.html.markdown

```diff
@@ -0,0 +1,128 @@
+---
+subcategory: "Cloud Platform"
+layout: "google"
+page_title: "Google: google_service_account_id_token"
+sidebar_current: "docs-google-service-account-id-token"
+description: |-
+  Produces OpenID Connect token for service accounts
+---
+
+# google\_service\_account\id\_token
+
+This data source provides a google OpenID Connect (`oidc`) `id_token` 
+
+For more information see
+[OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html#IDToken) as well as [Authenticating using Google OpenID Connect Tokens](https://github.com/salrashid123/google_id_token)
+
+## Example Usage
```

Comment:
It's probably worth thinking about how to be a bit more concise with the examples and descriptions of them. This is fairly wordy right now.